### PR TITLE
added brazilian portugese translation by @K0media (#894)

### PR DESCRIPTION
**Pull request by deanoemcke**
Pull request for Brazilian Portugese translation created by @K0media (#894)

### CODE REVIEW

**Comment by liamjohnston** [comment]
@deanoemcke To put my marketer's hat on for a moment, I'm not sold on this "O Grande Suspensor" bit... the extension's name is the extension's name. (Not to mention that the original gag is the pun (great pretender), which is lost.)

**Comment by liamjohnston** [comment]
Does le coq sportif translate their name in english speaking countries as "the athletic cock"? No. 

Though that would be awesome 😆

**Comment by liamjohnston** [comment]
It would also lead to a disparity/confusion with what the extension is called and referred to with their international peers. Also maybe bad for SEO...?

**Comment by deanoemcke** [comment]
lol. thanks for the input @liamjohnston 
see my comments here: https://github.com/deanoemcke/thegreatsuspender/issues/526

**Comment by liamjohnston** [comment]
Cool. Good move reverting I reckon (clearly) 👍
I'm not sure which comments you're specifically referring to in #526 though...?

**Comment by deanoemcke** [comment]
@liamjohnston oh, it's just that i said exactly the same thing as you over there about the extension title.

File: src/_locales/pt_BR/messages.json [hunk 1]
```diff
@@ -0,0 +1,755 @@
...
+    "message": "Recarregar guias selecionadas"
+  },
+  "ext_cmd_soft_suspend_active_window_description": {
+    "message": "Suspender todas as outras guias na janela ativa"
+  },
+  "ext_cmd_force_suspend_active_window_description": {
+    "message": "Forçar suspender todas as outras guias na janela ativa"
+  },
+  "ext_cmd_unsuspend_active_window_description": {
+    "message": "Resumir todas as guias na janela ativa"
```
Comment (K0media) [inline src/_locales/pt_BR/messages.json]: "message": "Recarregar todas as guias na janela ativa"

File: src/_locales/pt_BR/messages.json [hunk 2]
```diff
@@ -0,0 +1,755 @@
...
+    "message": "browserstack"
+  },
+  "html_about_browserstack_prefix": {
+    "message": "Obrigado também por"
+  },
+  "html_about_browserstack_suffix": {
+    "message": "disponibilizar ferramentas gratuitas de teste do Chrome."
+  },
+  "html_about_feeling_generous": {
+    "message": "Você pode apoiar O Grande Suspensor doando:"
```
Comment (K0media) [inline src/_locales/pt_BR/messages.json]: "message": "Você pode apoiar The Great Suspender doando:"

File: src/_locales/pt_BR/messages.json [hunk 3]
```diff
@@ -0,0 +1,755 @@
...
+    "message": "Sessões salvas"
+  },
+  "html_history_import_session": {
+    "message": "Importar sessão"
+  },
+  "html_notice_title": {
+    "message": "Aviso"
+  },
+  "html_notice_important_notice": {
+    "message": "Aviso importante do Grande Suspensor"
```
Comment (K0media) [inline src/_locales/pt_BR/messages.json]: "message": "Aviso importante do The Great Suspender"

File: src/_locales/pt_BR/messages.json [hunk 4]
```diff
@@ -0,0 +1,755 @@
...
+    "message": "Para colocar na lista branca vários sites em uma linha, você pode especificar parte da URL:"
+  },
+  "html_options_whitelist_tooltip_line3": {
+    "message": "Você também pode especificar expressões regulares colocando o texto em barras:"
+  },
+  "html_options_whitelist_test": {
+    "message": "Lista branca de teste"
+  },
+  "html_options_other_enable_context_menu": {
+    "message": "Adicionar o Grande Suspensor no menu de contexto do clique do botão direito do mouse"
```
Comment (K0media) [inline src/_locales/pt_BR/messages.json]: "message": "Adicionar The Great Suspender no menu de contexto do clique do botão direito do mouse"

File: src/_locales/pt_BR/messages.json [hunk 5]
```diff
@@ -0,0 +1,755 @@
...
+    "message": "Observação: a alteração dessa opção fará com que todas as suas guias suspensas desapareçam brevemente.\nNeste caso, agora pode ser um bom momento para fazer um backup da sua sessão de guias atuais."
+  },
+  "html_update_title": {
+    "message": "Atualização"
+  },
+  "html_update_heading2": {
+    "message": "está prestes a ficar maior"
+  },
+  "html_update_info": {
+    "message": "Uma nova versão do Grande Suspensor está pronta para ser instalada.\nObservação: durante a atualização, suas guias suspensas desaparecerão brevemente."
```
Comment (K0media) [inline src/_locales/pt_BR/messages.json]: "message": "Uma nova versão do The Great Suspender está pronta para ser instalada.\nObservação: durante a atualização, suas guias suspensas desaparecerão brevemente."

File: src/_locales/pt_BR/messages.json [hunk 6]
```diff
@@ -0,0 +1,755 @@
...
+    "message": "Esta extensão te faz feliz?\nMostre seu apoio doando."
+  },
+  "html_permissions_title": {
+    "message": "Permissões"
+  },
+  "html_permissions_heading": {
+    "message": "Por favor, me dê acesso a URLs de arquivos locais"
+  },
+  "html_permissions_info": {
+    "message": "No momento, esta extensão não é permitida a suspender as URLs que começam com a palavra \"file\".\nPara mudar isso, ative \"Permitir acesso a URLs de arquivo\" na página de configurações da extensão."
```
Comment (K0media) [inline src/_locales/pt_BR/messages.json]: "message": "No momento, esta extensão não está permitida a suspender as URLs que começam com a palavra \"file\".\nPara mudar isso, ative \"Permitir acesso a URLs de arquivo\" na página de configurações da extensão."

**Comment by K0media** [comment]
@deanoemcke, I noticed that you recently pushed a new release to the Webstore... Any news on this? A lot of my friends have been waiting for this feature to be implemented.

Would you please check if you can push the PT-BR localization publicly too?

Highly appreciated.

PS: I'm a bit worried about this since it has been almost a year I did this, but no returns yet. What have you been up to? 🤔

PSS: Sorry for the duplicate reply. I'm quite eager to see this rolled out in stable releases too.

**Comment by K0media** [comment]
Finally after a long while (1 year later) I see the translations in live. It took a dog's age to be rolled out, but I'm glad to finally see it there. Thank you so much for this!

Seriously. I've been expecting this for a long time now. Thanks for finally making it true.

Please tip me if there are new resources and features going to be available in future. I'll be glad to help here. This is such a handy extension, really. I'll do everything I can to help you guys out. 👍🏻

**Pull request merged**